### ISSUE TITLE
 aten::sub is not Supported!

### ISSUE THREAD

**Issue by qi657** [microsoft/nni#4636]
**Describe the issue**:
 **ERROR (nni.compression.pytorch.speedup.jit_translate/MainThread) aten::sub is not Supported! Please report an issue at https://github.com/microsoft/nni. Thanks~**

**Environment**:
- NNI version: 2.5
- Training service (local|remote|pai|aml|etc): local
- Client OS:
- Server OS (for remote mode only):
- Python version:
- PyTorch/TensorFlow version: 1.10.1
- Is conda/virtualenv/venv used?: conda/venv
- Is running in Docker?: no

**Configuration**:
 - Experiment config (remember to remove secrets!):
 - Search space:

**Log message**:
 - nnimanager.log:
 - dispatcher.log:
 - nnictl stdout and stderr:
 

**How to reproduce it?**: aten::sub is not Supported! Thanks!

**Comment by scarlett2018** [comment]
Thanks @qi657 for raising the issue. Add @J-shang to confirm whether this is a bug or known limitation.

**Comment by J-shang** [comment]
@qi657 , could you show more information about where you use `aten::sub` or which nn module you used cause this issue?

**Comment by qi657** [comment]
>In the process of pruning my project using the basic pruner and auto pruner methods. When updating the mask: 
![image](https://user-images.githubusercontent.com/52816600/158368933-62d61ddf-a941-4330-90a5-a6e51ab009ab.png)

**Comment by J-shang** [comment]
@qi657 , this pr should fix your issue, you can fix this in your source code for a workaround and this pr will merge in the next release, thanks for your report!

**Comment by qi657** [comment]
> @qi657 , this pr #4644 should fix your issue, you can fix this in your source code for a workaround and this pr will merge in the next release, thanks for your report!

@Contributor
[J-shang](https://github.com/J-shang) rsub also has the same problem.
ERROR (nni.compression.pytorch.speedup.jit_translate/MainThread) aten::rsub is not Supported! Please report an issue at https://github.com/microsoft/nni. Thanks~

**Comment by J-shang** [comment]
@qi657 , #4149 have fixed `rsub`, this will also merge in the next release.
